.github/workflows/test-python.yaml

```diff
@@ -50,3 +50,49 @@ jobs:
         run: |
           pip uninstall pandas -y
           python -c "import polars"
+
```

Comment:
Can you add a trigger that on this snippet that will also run when this yaml is changed?

```
on:
  pull_request:
    paths:
      - 'py-polars/**'
- 'polars/**'

```

Reply:
Maybe I am misunderstanding, but the same `on` clause is currently defined on lines 3 to 7, outside the `jobs`, so I assume it works for both? See also https://github.com/pola-rs/polars/runs/7436128231?check_suite_focus=true for the py3.7 run.

Reply:
No, I mean that we also trigger the job when the job definition itself has changed. Currently it is only triggered when python code has changed.

Reply:
Let me raise a separate PR for that.

Reply:
See #4125